### DISCUSSION TITLE
DBC float

### DISCUSSION THREAD

**Discussion by tyler206** [danielhrisca/asammdf#509] [Q&A]
Hi,
I have a question over using a DBC file with float signal. Does this software allow values with a float value to be converted in the Can Logging section?

For reference, I am using the line:
SIG_VALTYPE_ 1569 Speed : 1;
in dbc file.

Chris

**Comment by danielhrisca** [comment]
Hello Chris, floats should work. We use ``canmatrix`` for parsing the DBC files and I'm pretty sure this is supported
